### PR TITLE
Made changes in read me and added heading eddiehub

### LINKED ISSUE #4: add a heading EddieHub
add a Heading of `EddieHub` in the index.html file or make changes to the background image with the title eddiehub

### LINKED ISSUE #8: Make width to 100 % so the blur will not realised

### PR DESCRIPTION
**Pull request by rohank2502**
Added few screenshots and updated the read me file 
Fixes #8 
Fixes #4

### CODE REVIEW

**Comment by rohank2502** [comment]
It is looking like this !

**Comment by rohank2502** [comment]
Screenshot 2021-03-20 205809

File: index.html [hunk 1]
```diff
@@ -36,11 +36,12 @@
               <a href="https://discord.mlh.io/" target="_blank" class="sub-item">LHD: Share</a>
             </div>
           </div>
-          <div class="item"><a href="about.html"><i class="fas fa-info-circle"></i>About</a></div>
+          <div class="item"><a href="#"><i class="fas fa-info-circle"></i>About</a></div>
```
Comment (kcoder63) [inline index.html:39]: why did you remove about.html?

**Review by kcoder63 — CHANGES_REQUESTED** [review]
better to remove the screenshot with your email in it, we can keep it empty or crop it out?

**Comment by rohank2502** [comment]
>  to remove the screenshot with your email in it, we can keep it empty or crop it out?

I will make it blur will it be ok ?

File: index.html (hunk 1 above)
Comment (rohank2502) [reply index.html:39]: no I didnt by mistake might be will add it

**Comment by rohank2502** [comment]
> 
> 
> better to remove the screenshot with your email in it, we can keep it empty or crop it out?

Made those changes

File: README.md [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
+<h2 align="center">4. Select `EddieHub` in Guild Membership !!</h2>
+<img align="center" src="23.png" style="width:100%">
 
+<h2 align="center">4.Click on Submit !! You are done !!</h2>
+  
+  
+  
+<br>
+  
 Join EddieHub's discord server, where we geek out [here](https://discord.gg/PNHRZJf7v7)
```
Comment (Vyvy-vi) [inline README.md:17]:
```suggestion
Join EddieHub's discord server, where we geek out [here](http://discord.eddiehub.org)
```
Maybe using the permanent server link might be more manageable.

**Review by Vyvy-vi — COMMENTED** [review]
Maybe you can deploy the github pages site on your account, so that we could view it?

File: README.md (hunk 2 above)
Comment (rohank2502) [reply README.md:17]: I dont added that links

**Comment by rohank2502** [comment]
> 
> 
> Maybe you can deploy the github pages site on your account, so that we could view it?

https://rohank2502.github.io/Eddiehub_share/

**Review by SamDev98 — APPROVED** [review]
LGTM

**Comment by kcoder63** [comment]
> Maybe you can deploy the github pages site on your account, so that we could view it?

I did

**Review by kcoder63 — APPROVED** [review]
LGTM

**Pull request merged**